hubConf.py

```diff
@@ -0,0 +1,82 @@
+import cv2
+import torch
+import numpy as np
+import sys
+
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from baseline.utils.tensorboard import TensorBoard
+from baseline.Renderer.model import FCN
+from baseline.Renderer.stroke_gen import *
+
+from argparse import Namespace
+
+torch.manual_seed(1337)
+np.random.seed(1337)
+torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.benchmark = False
+use_cuda = torch.cuda.is_available()
+
+class Model:
+    def __init__(self, device=None, jit=False):
+        self.device = device
+        self.jit = jit
+        self.criterion = nn.MSELoss()
+        net = FCN()
+        self.step = 0
+        self.opt = Namespace(**{
+            'batch_size': 64,
+            'debug': '',
+            'script': False,
+            })
+
+        train_batch = []
+        ground_truth = []
+        for i in range(self.opt.batch_size):
+            f = np.random.uniform(0, 1, 10)
+            train_batch.append(f)
+            ground_truth.append(draw(f))
+
+        train_batch = torch.tensor(train_batch).float()
```

Comment:
did you need .float() here? maybe not an issue just not wondered if it was already float.

could also do .to(self.device) here, and avoid the 'if use_cuda' below

Reply:
When I removed the float, the model started complaining. So I did not remove it.

If I don't use 'use_cuda', it hits assertion errors when the GPU is not available, like in my case where I am using a devserver.

Reply:
ah- yea, that's good thinking and i guess our recipe isn't very well thought out.  But, i think all/most of our other models just assume cuda is available and don't do the check.  I'd also like to avoid the global 'use_cuda' since it kind of fails silently (if you run the __init__ with device==cuda, but your system doesn't have cuda, you'll think you're running cuda but actually run cpu). 

see this model for example.  for now, just follow this pattern and you won't be able to test the 'cuda' mode on your laptop but i'll verify it on my end, and you can also see that it works on the CI run.
https://github.com/ailzhang/attention-is-all-you-need-pytorch/blob/1f53f388829504b053199747f4767ab39d15edc2/hubconf.py#L71

Reply:
@wconstab : I agree Will. I have made changes as suggested above in the latest commit.